docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,572 @@
+---
+id: production-guide
+title: "Helm Chart Production Guide"
+sidebar_label: "Helm Chart Production Guide"
+description: "Learn how to set up the helm chart in a production setting."
+---
+
+## Overview
+
+This guide provides a simplified and streamlined architecture for core Camunda 8 applications using the Camunda Helm Chart. This setup minimizes complexity while offering a reliable foundation for most production use cases.
+
+The goal of this guide is to give you a **scenario-based, production focused, step-by-step guide** for setting up the Camunda Helm Chart. We will use AWS as a reference here but the guide would work with other supported cloud providers and their comparable services as well. By following this guide, you will be familiar with all of the necessary requirements for having a production ready Camunda Helm Chart.
+
+## Prerequisites
+
+Before proceeding with the setup, ensure the following requirements are met:
+
+- **Kubernetes Cluster**: A functioning Kubernetes cluster with kubectl access.
+- **Helm**: Helm CLI installed
+- **DNS Configuration**: Access to configure DNS for your domain to point to the Kubernetes cluster ingress.
+- **TLS Certificates**: Obtain valid X.509 certificates for your domain from a trusted Certificate Authority.
+- **External Dependencies**: Provision the following external dependencies:
+  - **Amazon Aurora PostgreSQL**: For persistent data storage required for the Web Modeler component.
+  - **Amazon OpenSearch**: For indexing and analytics.
+  - **AWS Simple Active Directory**: For authentication and authorization.
+- **NGINX Ingress Controller**: Ensure the NGINX ingress controller is set up in the cluster.
+- **AWS OpenSearch Snapshot Repository** - This will be a place to store the backups of the Camunda cluster. This repository must be configured with OpenSearch to take backups.
+- **s3 Bucket** - This will be used to store backups of the Camunda cluster. This s3 bucket must be configured with OpenSearch to take backups.
+- **Persistent Volumes**: Configure block storage persistent volumes for stateful components.
+- **Resource Planning**: Evaluate sufficient CPU, memory, and storage necessary for the deployment. Have a look at our [sizing guide](/components/best-practices/architecture/sizing-your-environment.md/#camunda-8-self-managed) for more information.
+
+Ensure all prerequisites are in place to avoid issues during installation or upgrading in a production environment.
+
+## Architecture Overview
+
+Below is the high-level architecture diagram for the base production setup _(click on the image to open the PDF version)_:
+[![Infrastructure Diagram ROSA Single-Region](./assets/smarch.jpg)](./assets/smarch.pdf)
+
+- Supported Components:
+  Camunda Platform Orchestration Core Components: Zeebe, Tasklist, Operate, Optimize
+- Ingress Controller (e.g., Nginx)
```

Comment:
here I'd again consider spelling out the one you mean to avoid confusion.
While I get that any Ingress controller would do the trick, the defaults, that afaik the guide is also relying on, is `ingress-nginx`.
Any other controller may require extra input from the user (not covered by Camunda)

Reply:
I will probably remove this section in favor of @hisImminence's comment here: https://github.com/camunda/camunda-docs/pull/4703#discussion_r1908657029

Reply:
I decided to remove the section under the diagram